README.md

```diff
@@ -41,102 +40,107 @@ pre-commit install
 
-### :eyeglasses: Pipeline
+### Tweaking the Visualization
+
+By now, there is no specific Kedro-style configuration.
+The generated plots can be adjusted using the `design` class located in [src/quafel/pipelines/visualization/nodes.py](src/quafel/pipelines/visualization/nodes.py).
+Propagating these settings to a `.yml` file is on the agenda!
+
+### Pipeline :eyeglasses:
 
 You can actually see what's going on by running
 ```
 poetry run kedro-viz
 ```
 which will open a browser with [kedro-viz](https://github.com/kedro-org/kedro-viz) showing the pipeline.
 
-![kedro-viz view of the pipeline](docs/kedro_view.png)
-
-### :floppy_disk: Data Structure
+## Data Structure :floppy_disk:
 
 - [data/01_raw](data/01_raw):
-  - [Evaluation Matrix](data/01_raw/dataset.json) containing all valid values for ```frameworks```, ```qubits```, ```depths```, and ```shots``` as specified in the [data_generation.yml](conf/base/parameters/data_generation.yml) file.
+  - **Versioned** [Evaluation Matrix](data/01_raw/dataset.json) containing all valid values for ```frameworks```, ```qubits```, ```depths```, and ```shots``` as specified in the [data_generation.yml](conf/base/parameters/data_generation.yml) file.
 - [data/02_intermediate](data/02_intermediate):
-  - Evaluation Partitions split into single ```.csv``` files
+  - Evaluation Partitions split into single ```.csv``` files.
+  - The number of partitions depend on the configuration.
 - [data/03_qasm_circuits](data/03_qasm_circuits/):
-  - as the name suggests, all generated qasm circuits for the job with the corresponding id
+  - A QASM circuit for each partition.
 - [data/04_execution_results](data/04_execution_results/):
-  - simulator results of the job with the corresponding id.
-  - result formats are unified as a dictionary with the keys containing the binary bit representation of the measured qubit and the normalized counts as values.
-  - results are zero padded, so it is ensured that also state combinations with $0$ probability are represented.
+  - Simulator results of the job with the corresponding id.
+  - Result formats are unified as a dictionary with the keys containing the binary bit representation of the measured qubit and the normalized counts as values.
+  - Results are zero padded, so it is ensured that also state combinations with $0$ probability are represented.
 - [data/05_execution_durations](data/05_execution_durations/):
-  - duration for the simulation of the job with the corresponding id.
-  - duration is only measured for the execution of the simulator
-  - combining and post-processing results (to obtain the dictionary representation) is not involved
+  - Duration for the simulation of the job with the corresponding id.
+  - Duration is only measured for the execution of the simulator
+  - Combining and post-processing results (to obtain the dictionary representation) is not involved
```

Comment:
What do you mean with involved? Isn't it about what is *contained* in those folders?